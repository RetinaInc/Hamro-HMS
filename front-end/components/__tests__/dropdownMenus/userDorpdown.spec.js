import React from 'react';
import {shallow, mount} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

import UserDropDown from 'components/dropdownMenus/userDropdown';
describe('UserDropDown', () => {
    it('renders correctly', () => {
        const userDropDown = shallow(<UserDropDown />);

        expect(shallowToJson(userDropDown)).toMatchSnapshot();


    });
});