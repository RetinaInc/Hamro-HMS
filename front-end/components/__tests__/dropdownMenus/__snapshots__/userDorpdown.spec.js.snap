// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDropDown renders correctly 1`] = `
<ul
  className="dropdown-menu dropdown-user"
>
  <li>
    <a
      href="#"
    >
      <i
        className="fa fa-user fa-fw"
      />
       User Profile
    </a>
  </li>
  <li>
    <a
      href="#"
    >
      <i
        className="fa fa-gear fa-fw"
      />
       Settings
    </a>
  </li>
  <li
    className="divider"
  />
  <li>
    <a
      href="/auth#/logout"
    >
      <i
        className="fa fa-sign-out fa-fw"
      />
       Logout
    </a>
  </li>
</ul>
`;
